requirements.txt

```diff
@@ -0,0 +1,21 @@
+cachetools==4.1.1
```

Comment:
How did you create this list?

Reply:
`python3 -m pip freeze > requirements.txt`

I think this is the standard way of generating it. I re-ran it and it had some updates that I included in the latest commit.